test/verify/check-journal

```diff
@@ -331,15 +331,15 @@ s.send("PRIORITY=3\\nFOO=bar\\n")'
         b.wait_present(sel)
 
     @skipImage("Newer version of ABRT required", "centos-7", "rhel-7", "fedora-25", "fedora-i386", "fedora-testing", "rhel-7-4")
-    @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1604", "rhel-atomic", "fedora-atomic")
+    @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1604", "rhel-atomic", "fedora-atomic", "continuous-atomic")
     def testAbrtDelete(self):
         b = self.browser
         m = self.machine
 
         self.login_and_go("/system/logs")
 
         m.execute("setenforce 0; ulimit -c unlimited; echo 'sleep 42m &' > slp; chmod u+x slp")
-        m.execute("./slp; killall -s SIGSEGV sleep")
+        m.execute("./slp; pkill -SEGV sleep")
```

Comment:
This simply was my first attempt to fix this before I realized that ABRT isn't available there. I don't think it hurts to keep this, both for always using the same CLI tool in tests and just in case we actually do add ABRT to atomic in the future.